lib/src/contributors/class/data_class_contributor.dart

```diff
@@ -85,6 +82,15 @@ class DataClassAssistContributor extends Object
 
     final ChangeBuilder changeBuilder = ChangeBuilder(session: session);
     await changeBuilder.addDartFileEdit(targetFilePath, (DartFileEditBuilder fileEditBuilder) {
+      // Shorthand constructor
```

Comment:
Remove these type of comments as the do not add any benefit, because the functiion name is descriptive enough